docs/api-testing/introduction.md

```diff
@@ -0,0 +1,59 @@
+
+If this contract is broken by either party, it can lead to bugs and malfunctions. _Contract testing_ is the act of validating that the API producer and the API consumer are respecting the contract. That’s where Sauce Labs API Testing comes in.
+
+## Contract Testing  
+
+Contract testing is a fast, lightweight form of API testing that strictly checks the content and format of requests and responses. This method is ideal for:
+* Testing APIs during the early stages of design and development
+* Organizations creating APIs that are internal and/or have limited number of consumers
+
+It can also validate spec files in an automated way by capturing how an API consumer and producer communicate with each other. This is typically done in a protected, static environment, where tests are run against mocked (not live) APIs, allowing contract tests to compare isolated API responses to the contract for immediate attention if something is wrong.<br/><img src={useBaseUrl('img/api-fortress/2022/03/api-consumer-contract.png')} alt="API Conversation and Contract" width="500"/>
+
+### API Server Side
+To test the producer (server) side:
+1. From an API Testing Project, go to the HTTP Client.
+2. Import an OpenAPI specification file (v3.0 or higher).
+3. From your list of **Snapshots**, choose the API call you'd like to test by clicking on it. The HTTP method, request URL, and anything else specified in the spec file will populate in the HTTP Client fields.
+4. Click **Generate Test**.
+
+Sauce Labs API Testing will validate the API producer side by creating a contract tests from your OpenAPI spec file.
+
+<img src={useBaseUrl('img/api-fortress/2022/03/api-producer-contract.png')} alt="API Conversation and Contract" width="500"/>
+
+### API Client Side
+To test the consumer (client) side:
+1. From an API Testing Project, go to the HTTP Client.
+2. Import an OpenAPI specification file (v3.0 or higher).
+3. Run [Piestry](/api-testing/mocking/), our API mocking server, with the same OpenAPI contract. You'll need to activate the contract testing functionality, binding with a Sauce Labs API Testing project.
+4. Run your unit tests against your client software, making sure the API URLs are pointing to the mocks provided by [Piestry](/api-testing/mocking/).
+
+Sauce Labs API Testing will validate that the API consumer side has complied with the contract specifications.
+
+<img src={useBaseUrl('img/api-fortress/2022/03/api-testing-overview.png')} alt="API Conversation and Contract" width="600"/>
+
+
+### Next Steps
+
+After you generate your test, you'll be taken to the **Compose** tool. Optionally, you can add further [assertions](/api-testing/composer/) to your test, which will perform [functional end-to-end testing](#functional-testing) on top of your contract tests and fully validate the APIs are working as intended.
+
+You can view your contract test's results and events on your [Project Dashboard](/api-testing/project-dashboard/).
+
+
+## Functional Testing
```

Comment:
@kimsauce Remove this part

Reply:
I think it might be good to show how functional testing relates to contract testing -- I'll change up this part and share with you all

Reply:
@kimsauce Functional Testing does not make sense here (I reviewed this with Simone and we both convey it does not make sense). Contract testing should be a single page not in the introduction